query/src/main/java/org/infinispan/query/clustered/ClusteredQueryOperation.java

```diff
@@ -38,12 +38,12 @@ public QueryDefinition getQueryDefinition() {
       return queryDefinition;
    }
 
-   static ClusteredQueryOperation createLazyIterator(QueryDefinition queryDefinition, UUID queryId) {
+   /*static ClusteredQueryOperation createLazyIterator(QueryDefinition queryDefinition, UUID queryId) {
```

Comment:
heads up, we've deprecated the whole "CacheQuery" class in 11, and this include the so called "lazy" iterator. So you could just get rid of it